src/e2e/resources/data/FeedbackRankOptionQuestionE2ESqlTest.json

```
@@ -0,0 +1,322 @@
+      "email": "Danny.e.tmms@gmail.tmt"
+    }
+  },
+  "accountRequests": {},
+  "courses": {
+    "course": {
+      "id": "tm.e2e.FRankOptQn.CS2104",
+      "name": "Programming Language Concepts",
+      "institute": "TEAMMATES Test Institute 1",
+      "timeZone": "Africa/Johannesburg"
+    },
+    "course2": {
+      "id": "tm.e2e.FRankOptQn.CS1101",
+      "name": "Programming Methodology",
+      "institute": "TEAMMATES Test Institute 1",
+      "timeZone": "Africa/Johannesburg"
+    }
+  },
+  "instructors": {
+    "instructor": {
+      "id": "00000000-0000-4000-8000-000000000501",
+      "account": {
+        "id": "00000000-0000-4000-8000-000000000001"
+      },
+      "name": "Teammates Test",
+      "email": "tmms.test@gmail.tmt",
+      "role": "INSTRUCTOR_PERMISSION_ROLE_COOWNER",
+      "isDisplayedToStudents": true,
+      "displayName": "Co-owner",
+      "privileges": {
+        "courseLevel": {
+          "canViewStudentInSections": true,
+          "canSubmitSessionInSections": true,
+          "canModifySessionCommentsInSections": true,
+          "canModifyCourse": true,
+          "canViewSessionInSections": true,
+          "canModifySession": true,
+          "canModifyStudent": true,
+          "canModifyInstructor": true
+        },
+        "sectionLevel": {},
+        "sessionLevel": {}
+      },
+      "course": {
+        "id": "tm.e2e.FRankOptQn.CS2104"
+      }
+    },
+    "instructor2": {
+      "id": "00000000-0000-4000-8000-000000000502",
+      "account": {
+        "id": "00000000-0000-4000-8000-000000000001"
+      },
+      "course": {
+        "id": "tm.e2e.FRankOptQn.CS1101"
+      },
+      "displayName": "Co-owner",
+      "email": "tmms.test@gmail.tmt",
+      "isDisplayedToStudents": true,
+      "name": "Teammates Test",
+      "privileges": {
+        "courseLevel": {
+          "canModifyCourse": true,
+          "canModifyInstructor": true,
+          "canModifySession": true,
+          "canModifySessionCommentsInSections": true,
+          "canModifyStudent": true,
+          "canSubmitSessionInSections": true,
+          "canViewSessionInSections": true,
+          "canViewStudentInSections": true
+        },
+        "sectionLevel": {
+        },
+        "sessionLevel": {
+        }
+      },
+      "role": "INSTRUCTOR_PERMISSION_ROLE_COOWNER"
+    }
+  },
+  "sections": {
+    "ProgrammingLanguageConceptsSection1": {
+      "id": "00000000-0000-4000-8000-000000000101",
+      "course": {
+        "id": "tm.e2e.FRankOptQn.CS2104"
+      },
+      "name": "Section 1"
+    }
+  },
+  "teams": {
+    "ProgrammingLanguageConceptsTeam1": {
+      "id": "00000000-0000-4000-8000-000000000201",
+      "section": {
+        "id": "00000000-0000-4000-8000-000000000101"
+      },
+      "name": "Team 1"
+    },
+    "ProgrammingLanguageConceptsTeam2": {
+      "id": "00000000-0000-4000-8000-000000000202",
+      "section": {
+        "id": "00000000-0000-4000-8000-000000000102"
```

```suggestion
        "id": "00000000-0000-4000-8000-000000000101"
```

the previous sectionId does not exist